networking/src/APIClients/datamanagers/CentralDataManagerAPIClient.ts

```diff
@@ -283,28 +294,45 @@ export class CentralDataManagerAPIClient implements ICentralDataManagerAPIClient
           this.authAPIClient.awsUploadImagePrivate(backImage) as any
         ]);
         updatedUser = {...updatedUser, frontIdPath: upadtingImages[0].path, backIdPath: upadtingImages[1].path}
-        console.log("Going to update images with these: ", upadtingImages)
+        console.log("Uploaded front image and back Image: ", upadtingImages)
       }
       catch (error: any) {
         throw new Error (JSON.stringify(error));
       }
     }
-
-    try {
-      if(newProfilePic){
-        const images = await Promise.all([
-          this.authAPIClient.awsUploadImagePrivate(newProfilePic) as any,
-        ]);
-        updatedUser = { ...updatedUser, profilePicPath: images[0].path};
-        console.log("After returning from s3!!", updatedUser)
+    if(frontImage || backImage){
```

Comment:
If we have some extra time after our internal QA this week, I bet we can write these so that they use Promise.all and run the authAPIClient calls concurrently (otherwise, these are going to await in series and take longer). Def just focus on Dev QA for now though